IDE/ECLIPSE/SIFIVE/Makefile

```diff
@@ -0,0 +1,38 @@
+PROGRAM ?= wolfcrypt
```

Comment:
None of the files in this SIFIVE directory are included in an .am file for distribution.  This has to be checked on every PR with new files.

Reply:
Added! Thanks